contrib/k8s/k8s-unmanaged.sh

```diff
@@ -3,8 +3,8 @@
 # Copyright Authors of Cilium
 
 function all_ceps { kubectl get cep --all-namespaces -o json | jq -r '.items[].metadata | .namespace + "/" + .name'; }
-function all_pods { kubectl get pods --all-namespaces -o json | jq -r '.items[] | select(.spec.hostNetwork==true | not) | .metadata | .namespace + "/" + .name'; }
+function all_pods { kubectl get pods --all-namespaces -o json | jq -r '.items[] | select((.status.phase=="Running" or .status.phase=="Pending") and (.spec.hostNetwork==true | not)) | .metadata | .namespace + "/" + .name'; }
  
-echo "Skipping pods with host networking enabled..."
+echo "Skipping pods with host networking enabled and with status not in Running or Pending phase..."
```

Comment:
```suggestion
echo "Skipping pods with host networking enabled or with status not in Running or Pending phase..."
```

Reply:
Makes sense, thanks! I have commited it.